pkg/controller/nodelifecycle/scheduler/timed_workers.go

```diff
@@ -52,11 +52,17 @@ type TimedWorker struct {
 // createWorker creates a TimedWorker that will execute `f` not earlier than `fireAt`.
 func createWorker(ctx context.Context, args *WorkArgs, createdAt time.Time, fireAt time.Time, f func(ctx context.Context, args *WorkArgs) error, clock clock.WithDelayedExecution) *TimedWorker {
 	delay := fireAt.Sub(createdAt)
+	fWrappedWithErrorLogging := func(ctx context.Context, args *WorkArgs) {
```

Comment:
simplification nit:

```suggestion
	fWithErrorLogging := func() {
```

the variables are already captured automatically.

Reply:
done